### PR TITLE
chore: add git attributes for generated files

### PR DESCRIPTION
**Pull request by weilbith**
The effect of these attributes are that they changes to these files do not generate big text diffs and also GitHub "ignores" them in pull requests.

Please note that the general goal is to get rid of all these files in the version control system on a long term.

### CODE REVIEW

**Comment by weilbith** [comment]
I'll push now a commit that modifies some of these generated files to test the effect on GitHub too. Locally it works as expected. I'll remove the commit later.

**Comment by weilbith** [comment]
I used the wrong linguist attribute... I'll force push directly instead of pushing a fixup commit.

**Comment by weilbith** [comment]
Alright, it seems to work now. Unfortunately does the line count still include those files their changes too. Would have been nice to get this affected too, to have reviewers see the actual size of the PR.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+poetry.lock binary linguist-generated
+deployments/**/* binary linguist-generated
```
Comment (istankovic) [inline .gitattributes:2]: Why are we ignoring everything under `deployments`?
Those files are tracked explicitly.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,2 @@
+poetry.lock binary linguist-generated
```
Comment (istankovic) [inline .gitattributes:1]: Why are we ignoring `poetry.lock`? We could, of course, just remove it completely from the repo and continue without it, but this is against best practice (https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control).

File: deployments/rinkeby/FillManager.json [hunk 3]
```diff
@@ -14069,4 +14085,4 @@
   "sourceMap": "392:2019:10:-:0;;;946:162;;;;;;;;;;;;;;;;;;;;;;;;;;;;:::i;:::-;921:32:0;719:10:6;921:18:0;:32::i;:::-;1018:10:10;:24;;-1:-1:-1;;;;;1018:24:10;;;-1:-1:-1;;;;;;1018:24:10;;;;;;;1052:14;:49;;;;;;;;;;;392:2019;;2270:187:0;2343:16;2362:6;;-1:-1:-1;;;;;2378:17:0;;;-1:-1:-1;;;;;;2378:17:0;;;;;;2410:40;;2362:6;;;;;;;2410:40;;2343:16;2410:40;2333:124;2270:187;:::o;14:177:21:-;93:13;;-1:-1:-1;;;;;135:31:21;;125:42;;115:70;;181:1;178;171:12;115:70;14:177;;;:::o;196:293::-;275:6;283;336:2;324:9;315:7;311:23;307:32;304:52;;;352:1;349;342:12;304:52;375:40;405:9;375:40;:::i;:::-;365:50;;434:49;479:2;468:9;464:18;434:49;:::i;:::-;424:59;;196:293;;;;;:::o;:::-;392:2019:10;;;;;;",
   "sourcePath": "contracts/contracts/FillManager.sol",
   "type": "contract"
-}
\ No newline at end of file
+}
```
Comment (istankovic) [inline deployments/rinkeby/FillManager.json:14088]: Please recreate the branch without these unneeded changes.

**Comment by weilbith** [comment]
@istankovic I think you generally misunderstand the purpose of this PR and what this is doing. Each of your comments if basically off. Especially if you read all my comments from above.
But let's don't waste your time with asynchronous communication. We will discuss this in person one day, as soon as the circumstances allow so.

**Comment by istankovic** [comment]
I don't think there is going to be more work on this so I am just going to close.

**Pull request closed without merging**